editor/filesystem_dock.cpp

```diff
@@ -2284,15 +2284,11 @@ Control *FileSystemDock::create_tooltip_for_path(const String &p_path) const {
 	}
 
 	const String type = ResourceLoader::get_resource_type(p_path);
-	Control *tooltip = nullptr;
+	Control *tooltip = EditorResourceTooltipPlugin::make_default_tooltip(p_path);
```

Comment:
Would it make sense to keep this default node around as a reference so we don't recreate it again below when plugins fail?

Reply:
tbh the code below is a leftover; it shouldn't be here anymore. If a plugin returns null, the editor will fallback to the default tooltip with filename.